portal/components/button/button.css

```diff
@@ -0,0 +1,62 @@
+/*
+The following ".button" global classes were created because the <Button>
+and <ButtonLink> components of "components/button.tsx" file cannot be used
+by the error pages "app/not-found.tsx" and "app/global-error.tsx". These pages would be rendered if the root layout
+crashes, so Adapters for Nuqs are not available. We don't need them, but the button.tsx component needs them for the ButtonLink component.
+Because of that, we can add all the styles here, and then use plain buttons in those error pages.
+*/
+.button--base {
```

Comment:
Is this following any known naming convention? It looks like BEM but is not quite that. If we can stick to a known standard, much better.

Reply:
I assumed `primary`, `secondary`, and `tertiary` to be all variants. As well as the sizes (both icons and regular). None of them impacts any other child element. Because of this, I thought all of them would be modifiers, hence the usage of `--`